frontend/app/views/fragments/event/headerBarCovid.scala.html

```diff
@@ -4,7 +4,8 @@
     <h2 class="header-bar__title">
         Given the growing concerns over public health and safety due to the developing Coronavirus (COVID-19)
         situation, we have suspended our events programme until 31 May 2020. For updates please follow us on
-        <a href="@twitterLink">Twitter</a>
-        or <a href="https://www.theguardian.com/guardian-masterclasses/2015/jan/19/sign-up-to-the-guardian-masterclasses-newsletter">sign up to our newsletter</a>.
+        <a href="@twitterLink">Twitter</a>,
+        <a href="https://www.theguardian.com/guardian-masterclasses/2015/jan/19/sign-up-to-the-guardian-masterclasses-newsletter">sign up to our newsletter</a>,
+        or express your interest in future events <a href="https://guardiannewsampampmedia.formstack.com/forms/masterclasses__express_your_interest_in_this_course">here</a>.
```

Comment:
Apologies if the card wasn't too clear on this, but I think they want this only on the masterclasses page - so we might have to make this part conditional or pass it into this component?